babel.config.cjs

```diff
@@ -14,7 +14,12 @@ module.exports = {
                 modules: process.env.NODE_ENV === "test" ? "commonjs" : false,
             },
         ],
-        "@babel/preset-typescript",
+        [
+            "@babel/preset-typescript",
+            {
+                rewriteImportExtensions: process.env.NODE_ENV !== "test",
```

Comment:
might be good to have a comment here explaining why we set this at all, and why we *don't* set it for jest.

Reply:
Good idea. Have added one.